RELEASE.md

```diff
@@ -49,3 +49,22 @@ just release-testflight
 ```
 
 Once uploaded, log into `appstoreconnect.apple.com/apps/` and approve testing the new version.
+
+## Using fastlane
+
+Make sure that all environment variables are set in the `.env` file.
+
+You will also need to install [`fastlane`](https://fastlane.tools/).
+
+1. We need to build the IPA file first without code signing because fastlane does not support `--dart-define`.
```

Comment:
This is quite unfortunate. An alternative solution would be to manually edit the files which are being edited if `--dart-define` is being used. See answer here: https://stackoverflow.com/questions/64396002/flutter-fastlane-ios-how-to-pass-dart-define

Reply:
I guess we can try living with it for now....if the overhead becomes too big, we can always try the other way.

am I correct in thinking that these IPA files can also be used to upload to a phone and test that way?